maven/impl-maven/src/main/java/org/jboss/shrinkwrap/resolver/impl/maven/bootstrap/MavenRepositorySystem.java

```diff
@@ -52,13 +55,13 @@
  */
 public class MavenRepositorySystem {
 
-    private final RepositorySystem system;
+    private final Context context;
```

Comment:
Not done yet, MavenRepositorySystem should be made Closeable (as resolver requires clean shut down)...